### PR TITLE
Added dropdownscrollbar for a large list.

### PR DESCRIPTION
**Pull request by vasco-santos**
When it was used big lists, the dropdown had the size of the all list. Now it has a maximum size and it is possible to scroll the list.

### CODE REVIEW

File: angular-dropdowns.css [hunk 1]
```diff
@@ -56,6 +57,8 @@
   font-weight: normal;
   list-style: none;
   visibility: hidden;
+  max-height: 300px;
+  overflow-y: scroll; /*Provide an auto overflow to display scroll*/
```
Comment (jseppi) [inline angular-dropdowns.css]: I think `auto`, would be better

File: dist/angular-dropdowns.css [hunk 2]
```diff
@@ -56,6 +57,8 @@
   font-weight: normal;
   list-style: none;
   visibility: hidden;
+  max-height: 300px;
+  overflow-y: scroll; /*Provide an auto overflow to display scroll*/
```
Comment (jseppi) [inline dist/angular-dropdowns.css]: I think `auto`, would be better

**Comment by vasco-santos** [comment]
Yes, Auto is a better option, I completely forgot to change it.
Commited new version

**Comment by jseppi** [comment]
thanks!

**Pull request merged**